test/dsl_test.go

```diff
@@ -55,21 +58,51 @@ func test(t testing.TB, actions ...optionOp) {
 	}
 }
 
+func parallel(actions ...optionOp) optionOp {
+	return func(o *opt) {
+		wg := new(sync.WaitGroup)
```

Comment:
Hmm... with a value type, doesn't it allow accidentally copying the `wg` object? All the methods defined on `WaitGroup` are pointer types and it looks like [there'd be issues copying it](https://golang.org/src/sync/waitgroup.go?s=522:830#L8).